.gitignore

```diff
@@ -1,3 +1,9 @@
+# XSCEN-specific
+paths.yml
+# Files generated by notebooks
+*.zarr
+*.nc
```

Comment:
```suggestion
```

Reply:
I think we want to keep those there for now? We can delete those lines when/if we need datsets for the documentation.

Reply:
Files generated by the notebooks will be within the notebooks submodule (and thus managed by the submodule's gitignore), so these two lines could be erased. I'll do it in my commits!